scripts/protocgen.sh

```diff
@@ -15,6 +15,10 @@ protoc_gen_doc() {
   go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc 2>/dev/null
 }
 
+protoc_gen_gateway2() {
+  go get github.com/aleem1314/grpc-gateway/protoc-gen-grpc-gateway2@v1.16.5-final 2>/dev/null
```

Comment:
Thanks @aleem1314 for taking a look! I'm not sure it's worth to fork grpc-gateway. Maybe it's better to just remove our customer codegen for now.